src/main/resources/com/google/api/codegen/java/initcode.snip

```diff
@@ -2,9 +2,10 @@
 
 @snippet initCode(initCodeSpec)
   @if initCodeSpec.argDefaultLines
-    // FIXME(pongad): should be commented to not clobber args
-    {@initCodeLines(initCodeSpec.argDefaultLines)}
-    // FIXME(pongad): args end
+    @join line : util.pretty(initCodeLines(initCodeSpec.argDefaultLines))
+      // {@line}
+    @end
+
```

Comment:
I have no idea why these newlines are required, but they are. Otherwise we get really weird results like
```
// this the last line of the commentThis should be the first line but somehow it is concatenated here.
```

Reply:
Did you try `@join ... on BREAK`?

Reply:
I did, but I don't think it can possibly matter, if you don't have the `on` clause, it joins on BREAK by default: https://github.com/googleapis/api-compiler/blob/093283bb4bf19f3866bca5e784fe722d277afbc5/src/main/java/com/google/api/tools/framework/snippet/SnippetParser.java#L306